.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,61 +1,53 @@
 ---
 name: 🐛 Bug report
 about: Create a report to help us improve
-
 ---
-
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
+<!--- Verify first that your issue is not already reported on GitHub -->
+<!--- Also test if the latest release and devel branch are affected too -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
```

Comment:
What is too invisible ?

Reply:
lowercase warning in the end about the bot. I couldn't visually catch it instantly even though I've been looking for it. A normal reader would just skip-through this line while scanning.

Reply:
We reduced the number of messages, so that the user would read more. I think all three statements are important to have. The two statements prior to this are even more relevant to the act of opening an issue. If this one would stand out (e.g. in capital) that's the only thing most people would take away...

Reply:
If rephrasing that sentence could help, proposals are welcomed.

Reply:
Can we put :robot: emoji there? :)

`🤖`

Reply:
I'd prefer if we didn't

Reply:
I rephrased the sentence, maybe it's better now ?